views/index.hbs

```diff
@@ -29,7 +29,7 @@
           <div class="one column row">   
           </div> 
           <div class="one column row">
-              <div class="center aligned sixteen wide column"><button class="ui purple basic button massive">Enter in ChatRoom</button></div>
+              <div class="center aligned sixteen wide column"><button onclick="location.href = '/chat';" class="ui purple basic button massive">Enter in ChatRoom</button></div>
```

Comment:
rather than this. make it an anchor tag, and add the link to href.

Reply:
Alright.